modules/developer_manual/pages/testing/acceptance-tests.adoc

```diff
@@ -255,6 +255,30 @@ To do that, specify `SHOW_OC_LOGS=true`:
 make test-acceptance-api BEHAT_SUITE=apiTags SHOW_OC_LOGS=true
 ----
 
+=== Get Detailed Information About API Requests
+
+
```

Comment:
only one blank line necessary.
(cant do a commit suggestion for just removing a blank line...)

Reply:
I saw that the other sections in this doc have 2 blank lines, so I thought there must be a reason and copied.